bedrock/firefox/templates/firefox/includes/release-menu.html

```diff
@@ -1,11 +1,15 @@
 <nav id="nav-main" role="navigation">
   <span id="nav-main-toggle" class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
   <ul id="nav-main-menu">
-    <li class="first"><a class="more" rel="external" href="https://developer.mozilla.org">{{_('MDN')}}</a>
-    </li>
-    <li><a href="{{ url('firefox') }}">{{_('Firefox')}}</a>
-    </li>
-    <li><a href="{{ url('firefox.developer') }}">{{_('Firefox Developer Edition')}}</a>
-    </li>
+    <li class="first"><a class="more" rel="external" href="https://developer.mozilla.org">{{_('MDN')}}</a></li>
+    <li><a href="{{ url('firefox') }}">{{_('Firefox')}}</a></li>
+    {% if release.product == 'Firefox for Android' %}
+    <li><a href="{{ url('firefox.channel.android') }}">{{_('Firefox Nightly &amp; Beta')}}</a></li>
+    {% elif release.product == 'Firefox for iOS' %}
+    <li><a href="{{ url('firefox.channel.ios') }}">{{_('Firefox Beta')}}</a></li>
+    {% else %}
+    <li><a href="{{ url('firefox.channel.desktop') }}">{{_('Firefox Nightly &amp; Beta')}}</a></li>
```

Comment:
👍 to combine "Firefox Nightly & Beta" with "Firefox Developer Edition" into "Pre-release builds" or "Pre-release builds" (the latter label is used in the newer nav on [/firefox/desktop/](https://www.mozilla.org/en-US/firefox/desktop/) etc.